packages/backend/src/app.ts

```diff
@@ -295,20 +303,22 @@ export class App {
       })
   }
 
-  private async processInvoice(): Promise<void> {
-    const invoiceService = await this.container.use('invoiceService')
-    return invoiceService
+  private async processIncomingPayment(): Promise<void> {
+    const incomingPaymentService = await this.container.use(
+      'incomingPaymentService'
+    )
+    return incomingPaymentService
       .processNext()
       .catch((err) => {
-        this.logger.warn({ error: err.message }, 'processInvoice error')
+        this.logger.warn({ error: err.message }, 'processIncomingPayment error')
         return true
       })
       .then((hasMoreWork) => {
```

Comment:
I don't know why these changes prompted those build errors (or why similar worker functions aren't a problem), but this seems to fix it:
```
      .catch((err: Error) => {
        this.logger.warn({ error: err.message }, 'processIncomingPayment error')
        return true
      })
      .then((hasMoreWork: boolean) => {
```